source/jormungandr/tests/graphical_isochrons_tests.py

```diff
@@ -85,6 +85,56 @@ def test_graphical_isochrons_to_stop_point(self):
         assert (multi_poly.contains(stopA))
         is_valid_graphical_isochron(response, self.tester, q)
 
+    def test_graphical_isochrons_no_datetime(self):
+        q = "v1/coverage/main_routing_test/isochrons?from={}&duration={}&_current_datetime={}"
+        q = q.format(s_coord, '3600', '20120614T080000')
+        response = self.query(q)
+        query = "v1/coverage/main_routing_test/isochrons?from={}&datetime={}&duration={}"
+        query = query.format(s_coord, '20120614T080000', '3600')
+        resp = self.query(query)
+
+        is_valid_graphical_isochron(response, self.tester, q)
+        assert len(response['isochrons']) == len(resp['isochrons'])
+
+        for i in range(0, len(response['isochrons'])):
```

Comment:
a more pythonic way to write it would be to use `zip`

something like: `for isochron_no_dt, isochron in zip(response_no_dt, isochron):`